File: Beginner/1020 (Age in Days)/1020.cpp
#include <iostream>
using namespace std;

int main()
{
   int I;
   cin >> I;
   cout << I/365 << " ano(s)" << endl; I=I%365;
   cout << I/30 << " mes(es)" << endl; I=I%30;
   cout << I << " dia(s)" << endl;

    return 0;
}

File: Beginner/1020 (Age in Days)/1020.java
import java.io.IOException;
import java.util.Scanner;


public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);

        int n = ler.nextInt();
        
        System.out.println(n/365 + " ano(s)"); n = n %365;
        System.out.println(n/30 + " mes(es)"); n = n %30;
        System.out.println(n + " dia(s)");
    }
 
}